times_excel_reader.py

```diff
@@ -1470,6 +1476,8 @@ def get_matching_commodities(row):
                         df["VALUE"] = [row.VALUE] * len(df)
                     if len(df) == 0:
                         print(f"WARNING: {tag} row matched nothing")
+                    # for speed, extract just the VALUE column as that is the only one being updated
+                    df = df[["VALUE"]]
```

Comment:
You could do this on line 1463 for even more efficiency

Reply:
Done. Thanks